test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test 'should get index' do
+    get users_index_url
+    assert_response :success
+  end
+
+  test 'should get show' do
+    get users_show_url
+    assert_response :success
+  end
+end
```

Comment:
- [ ] Same for User specs what I have mentioned the change in posts spec.

Reply:
Hello @Deepakdanger , thanks for your feedback , they are all resolved